.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,15 @@
+name: golangci-lint
+on:
+  pull_request:
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: golangci-lint
```

Comment:
For now, we should just run staticcheck, as previously. To run the full lint collection, we need to fix many inconsistencies that got accumulated in time.